js/widgets/widgetWindows.js

```diff
@@ -64,26 +64,7 @@ function WidgetWindow(key, title) {
     // Needed to keep things canvas-relative
     let canvas = docById("myCanvas");
 
-    // Scrolling in window body .
```

Comment:
You cannot just delete this code as widgetWindows is used by **all** of the widgets, not just musickeyboard.

Reply:
> * [ ]  Disable the scrolling that is enabled in windowWidget (one simple way to do this would be to pass in a parameter),
I actually took this literally, sorry will think of an alternative to this one and update soon